unittests/data/type_traits_castxml.hpp

```diff
@@ -195,8 +195,8 @@ namespace no{
     typedef std::set< std::string > string_set_type;
     typedef std::multimap< std::string, std::string > s2s_multimap_type;
     typedef detail::vertex vertex_type;
-    typedef detail::const_item const_item_t;
-    typedef detail::const_container const_container_t;
+    // typedef detail::const_item const_item_t;
```

Comment:
I'm glad you looked up the history. That second commit seems to indicate you had to move those tests to the `no` namespace because `is_const` wasn't behaving correctly. I'll take a look into combining the two headers into one tonight and update the PR.